examples/object_detection/layers/modules/l2norm.py

```diff
@@ -16,8 +16,14 @@
 import torch.nn.init as init
 
 from nncf.utils import add_domain
+from nncf import register_module
 
+
+@register_module
 class L2Norm(nn.Module):
+
+    filter_algorithms = ['filter_pruning']
```

Comment:
Should instead be supplied as an argument to the `@register_module` invocation.
`@register_module(ignored_algorithms=['filter_pruning'])`

Reply:
This is related to the problem of user modules having weights in attributes that are named other than `weight`. While you're at it, you may also lay down another argument to `@register_module` that will allow the user to specify just that - the attribute name(s) of the module's weight(s).